source/web_integration/pnapi_web_integration.rst

```diff
@@ -95,11 +128,11 @@ Client Side Parameters
       Set of JavaScript parameters to be captured where the key name should be RingPool parameter name.
       These values are updated every subsequent visit.
 
-  * - savePoolParams
+  * - updatePoolParams
     - Bool
-    - Default: `false`
+    - Default: `true`
 
-      When true, the integration code will not update `poolParams` on subsequent page visits.
+      When `false`, the integration code will not update `poolParams` on subsequent page visits.
```

Comment:
Are we actually changing parameter names and flipping the contract in the same version? Is `2016-08-01` available to customers already?